flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -87,6 +101,17 @@ function setup_kubernetes_for_linux {
     sudo sysctl fs.protected_regular=0
 }
 
+function download_circtl {
+    local arch=$(get_arch)
+    local crictl_version crictl_archive arch
+    crictl_version="v1.24.2"
```

Comment:
Moving the version declaration further away from the Minikube installation might not be what we want. Because the versions are depending on each other: If we need to change the Minikube version, we might also need to change `crictl` and/or the `cri-dockerd` version. Having such variables spread over a script (or even different scripts) increases the chance on missing a version.

So in this sense, it might make sense to reconsider the code reorg here. Maybe, we should collect the related version variables next to `MINIKUBE_VERSION` in the header of the script. WDYT?